src/test/java/org/elasticsearch/test/ElasticsearchIntegrationTest.java

```diff
@@ -227,9 +226,25 @@
  * </p>
  */
 @Ignore
-@AbstractRandomizedTest.Integration
+@ElasticsearchIntegrationTest.Integration
+@LuceneTestCase.SuppressFileSystems("ExtrasFS") // doesn't work with potential multi data path from test cluster yet
 public abstract class ElasticsearchIntegrationTest extends ElasticsearchTestCase {
```

Comment:
Lets do it. i wanted to avoid it to lower the noise, but i don't like the inconsistency either.